src/session/mod.rs

```diff
@@ -136,7 +138,7 @@ mod imp {
                         "user-list",
                         "User List",
                         "The list of users of this session",
-                        ChatList::static_type(),
+                        UserList::static_type(),
```

Comment:
This could also be fixed in a separate Commit or PR.

Reply:
Done in #218.